### ISSUE TITLE
Convert analytics opt out feature to a button instead of a link

### ISSUE THREAD

**Issue by meganesu** [meganesu/personal-site-v2#23]
Currently, the way I've implemented opting out of Google Analytics for my site involves a script that runs when a user clicks a link. Ideally, that should be a button instead.

By the end of this issue, this lint error should not longer be appearing:

```
warn ESLintError:
C:\Users\Megan Sullivan\Documents\github-projects\personal-site-v2\src\pages\privacy.js
  41:7   warning  The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md
jsx-a11y/anchor-is-valid
  41:15  warning  Script URL is a form of eval
                                                                                                                                                                                  no-script-url 
```

**Comment by meganesu** [comment]
Closed by acc01eaec9c2517834f9cd45e417efb6bd9e32b5
